### ISSUE TITLE
[Accessibility- Images C 1.2] Set a suitable alt to images of published article 

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/meeds#3363]
**Environment:** 7.x

**Step of reproduction:**

- Create an article with image
- Publish it in the stream page and in a News list

**Current behavior:**

- The image of the published article in the stream has alt=" "
- The image of the published article in the News list has alt=" "

**Expected behavior:**

-  If **an alt is added** to the article image:

1. The image of the published article in the stream must have alt="[the added alt] "
2. The image of the published article in the News list must have alt="[the added alt] "

-  if **no alt is added** to the article image

1. The image of the published article in the stream must have alt=" "
2. The image of the published article in the News list must have alt=" "
